pkg/minikube/bootstrapper/kubeadm/kubeadm.go

```diff
@@ -526,9 +539,10 @@ func (k *Bootstrapper) PullImages(k8s config.KubernetesConfig) error {
 		return fmt.Errorf("pull command is not supported by kubeadm v%s", version)
 	}
 
-	cmd := fmt.Sprintf("%s config images pull --config %s", invokeKubeadm(k8s.KubernetesVersion), yamlConfigPath)
-	if err := k.c.Run(cmd); err != nil {
-		return errors.Wrapf(err, "running cmd: %s", cmd)
+	cmd := command.ExecCmd(fmt.Sprintf("%s config images pull --config %s", invokeKubeadm(k8s.KubernetesVersion), yamlConfigPath))
+	rr, err := k.c.RunCmd(cmd)
+	if err != nil {
```

Comment:
```suggestion
	if rr, err := k.c.RunCmd(cmd); err != nil {
```

there are a few places I feel that writing like ^^ will improve readability, but this is nitpick